code/game/turfs/open/_open.dm

```diff
@@ -235,7 +235,7 @@
 			C.Paralyze(paralyze_amount)
 			C.stop_pulling()
 		else
-			C.Knockdown(20)
+			C.Paralyze(20)
```

Comment:
?

Reply:
Goof are you trying to sneak in slip stuns again?

Reply:
Knockdown and Paralyze are both stuns, Paralyze is supposed to not visually knock you down. It's so that you visually slide while standing up, but this seems to be broken right now.

Reply:
Er no.
Knockdown is when you're stuck on the ground like after a slip but aren't actually stunned.
Paralysis is on the ground and stunned.

Reply:
Paralyze is the one that knocks you to the floor and stuns you. Stun() is the effect used by paralytics such as curare.

Reply:
resolved, it uses Immobilize() now